test/verify/check-system-services

```diff
@@ -208,6 +208,22 @@ Description=Test OnBoot Timer
 [Timer]
 OnBootSec=200min
 Unit=test.service
+""")
+        self.write_file("/usr/local/lib/systemd/system/cockpit-system.timer",
+                        """
+[Unit]
+Description=Not deleteable Timer
+
+[Timer]
+OnCalendar=*:1/2
+""")
+        self.write_file(f"{path}/deleteme.timer",
```

Comment:
This should also write a corresponding deleteme.service, and ensure that it gets cleaned up.